ceph_deploy/hosts/remotes.py

```diff
@@ -179,10 +187,11 @@ def listdir(path):
     return os.listdir(path)
 
 
-def makedir(path, ignored=None):
+def makedir(path, ignored=None, uid=-1, gid=-1):
     ignored = ignored or []
     try:
         os.makedirs(path)
+        os.chown(path, uid, gid);
```

Comment:
Personally I would have it independent of the except but my wording was unclear, but the else is fine too. Is os.chown() recursive? I did not think so. and looking at the docs it makes no specific mention of this. https://docs.python.org/2/library/os.html